packages/react/src/components/StorybookSnapshots.test.js

```diff
@@ -97,6 +97,7 @@ describe(`Storybook Snapshot tests and console checks`, () => {
   });
   initStoryshots({
     storyKindRegex: /Watson\sIoT.*$|.*Getting\sStarted/g,
+    storyNameRegex: /^((?!.*?DontTest).)*$/,
```

Comment:
Whats this for?

Reply:
This is a relic that was a way to get around some test failures, letting the suite pass while skipping a few stories. I'm trying to find the root cause instead